content/sensu-go/6.2/operations/monitoring-as-code/_index.md

```diff
@@ -0,0 +1,338 @@
+Teams can share and remix monitoring workflows that include Sensu resources for collecting observability events and metrics, diagnosing issues, sending alerts, and automatically remediating problems.
+
+In the monitoring as code approach, when a new endpoint starts up, like a cloud compute instance or Kubernetes Pod, Sensu automatically registers itself with the platform and starts collecting monitoring and observability data according to the code in your configuration files.
+Your configuration files are composed of the Sensu resource definitions you use in your monitoring and observability workflows.
+You can manage your monitoring and observability workflow as code, the same way you manage other code and the systems you're monitoring, to improve visibility, reliability, portability, and repeatability.
+
+- Share, edit, review, and version your Sensu workflow configuration files just like you would with other "as code" solutions, within one team or among teams across your organization.
+- Maintain revision control and change history for your monitoring workflows.
+- Export your Sensu configuration from one environment and initialize another environment with the same configuration.
+- Remove, restore, back up, and recover Sensu instances based on Sensu workflow configuration files.
+- Use `sensuctl prune` to consistently apply changes to your monitoring workflows with a single command.
+- Include your monitoring workflow in your centralized continuous integration/continuous delivery (CI/CD) pipeline to keep your monitoring aligned with your product and services.
+
+To get started with monitoring as code, you'll need a [repository][11] and [configuration files][13] that contain your resource definitions.
+
+## Create a monitoring as code repository
+
+A monitoring as code repository will include configuration files that contain the resource definitions you use in your monitoring workflow.
+You can use any source control repository for maintaining your monitoring workflow configuration files, but you'll need to decide how to structure your repository.
+
+The way you will use your Sensu resource [configuration files][13] will help you choose the best structure for your monitoring as code repository:
+
+- If you want to share complete end-to-end Sensu workflows with your colleagues, you might save all of the resource definitions for each workflow in a single configuration file.
+This allows others to read through an entire workflow without interruption.
+- If you are more likely to share individual workflow components, it makes more sense to use individual configuration files for different types of resources, like one file for all of your checks, one file for all of your handlers, and so on.
+- If you want to facilitate even more granular sharing, you can save one resource definition per file.
+
```

Comment:
Just occurred to me -- maybe we want to say something about considering what's best if you're going to eventually manage these via CI/CD.